sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/package.scala

```diff
@@ -132,7 +133,7 @@ package object util extends Logging {
     val ret = f
     val endTime = System.nanoTime()
     // scalastyle:off println
-    println(s"${(endTime - startTime).toDouble / 1000000}ms")
+    println(s"${(endTime - startTime).toDouble / TimeUnit.MILLISECONDS.toNanos(1)} ms")
```

Comment:
Just convert the whole difference from nanos to ms?

Reply:
This method `benchmark` is not used at all. Maybe we remove it?

Reply:
Yeah, just remove it

Reply:
removed